R/util.R

```diff
@@ -142,3 +142,35 @@ lookupByName = function(listpath, name, prop = "name") {
   assign(key, val, envir = byNameHash)
   val
 }
+
+#' Find a Resource By Name
+#'
+#' Allows createion of a by-experiment lookup object, which can
+#' then be used to find resources by name instead of id. Utilizes
+#' the internal lookupByName cache to improve performance.
```

Comment:
This is an implementation detail we don't need to reveal to users. The next sentence is sufficient.